### ISSUE TITLE
Accessibility: Screen reader no longer reads text in Paragraph block when editing

### ISSUE THREAD

**Issue by jamiepalatnik** [Automattic/wp-calypso#51408]
### Steps to reproduce the behavior

1. Using JAWS screen reader, go to a Paragraph block with text to edit.
2. Edit text.

#### What I expected to happen

Screen reader should read out text letter by letter so that user can edit. This allows the user to correct typos and make other changes.

#### What actually happened

The cursor moves as expected but the screen reader only reads the first letter and does not read the other letters.

### Context

3853396-zd-woothemes<br>#user-report

Wondering if this issue could be related: [https://github.com/WordPress/gutenberg/issues/29526](<https://github.com/WordPress/gutenberg/issues/29526>)

**Comment by daria2303** [comment]
3964395-zd-woothemes

**Comment by autumnfjeld** [comment]
See [Editable blocks have inconsistent ARIA role: sometimes textbox, sometimes document](https://github.com/WordPress/gutenberg/issues/42158#top)

It looks like this is a Gutenberg issue.
